docs/multicluster/architecture.md

```diff
@@ -34,33 +36,42 @@ given ClusterSet.
 
 ## Antrea Multi-cluster Controller
 
-In a member cluster, Antrea Multi-cluster creates a Deployment that runs Antrea Multi-cluster
-Controller which is responsible for exporting resource to and importing resource from a leader
-cluster in a ClusterSet.
+Antrea Multi-cluster Controller watches ClusterSet, ClutserClaim, Service, ServiceExport etc.
+It has different responsibilities in member and leader clusters.
 
-In a leader cluster, Antrea Multi-cluster creates a Deployment that runs Antrea Multi-cluster
-Controller which is responsible for converting resources from different member clusters into one
-encapsulated resource as long as these resources have the same kind and match Namespace sameness.
+In a member cluster, Antrea Multi-cluster creates a Deployment with one replica that runs
+Antrea Multi-cluster Controller which is responsible for exporting resource to and importing
+resource from a leader cluster in a ClusterSet.
 
-In ClusterSet initialization, Antrea Multi-cluster Controller in a member cluster watches
+In a leader cluster, Antrea Multi-cluster creates a Deployment with one replica that runs
+Antrea Multi-cluster Controller which is responsible for converting resources from different
+member clusters into one encapsulated resource as long as these resources have the same kind
+and match Namespace sameness.
+
+When ClusterSet is initialized in member cluster, Antrea Multi-cluster Controller watches
 ClusterSet, ClusterClaim, and creates a MemberClusterAnnounce in the leader cluster.
 
-For exporting resources from the member cluster, it watches ServiceExport, Service and Endpoints
-resources, encapsulates Services and Endpoints into ResourceExports according to ServiceExports,
-and writes the ResourceExports to leader cluster. For resource importing, it watches ResourceImports
-from leader cluster, and creates multi-cluster Services and Endpoints with a prefix `antrea-mc-`
-plus exported Service name, and also ServiceImports which have the same name as the Service name.
+For resources exporting in the member cluster, Antrea Multi-cluster Controller watches ServiceExport,
```

Comment:
This section is not well organized. I feel harder to explain all my suggestion than just writing down what is in my mind. Probably let me push a version for you to review.